docs/core/config/auth/passdb.md

```diff
@@ -442,6 +445,44 @@ a load spike of everybody getting logged in at exactly the same time.
 
 Do not perform any authentication, just store extra fields if user is found.
 
+##### `check_client_fp`
+
+Match client certificate or public key fingerprint.
+
+Requires configuring [[setting,ssl_peer_certificate_fingerprint_hash]], [[setting,auth_ssl_require_client_cert]],
+and [[setting,ssl_server_request_client_cert]] to require a client cert.
+
+This is intended to replace CA certificates with verifying client certificates using fingerprints, or to enforce
+that particular public key is being used. If you are using CA certificates,
+you can only fail authentication if the value is non-empty and does not match. If CA certificates are not used, then
+if no passdb matches the fingerprint, then authentication is failed.
```

Comment:
Is this saying that if CA certificates are used, the failure is not per-passdb? It's not very clear which parts of the text apply to CA usage and non-CA usage here.